lib/installed_clients/ReadsAPIClient.py

```diff
@@ -25,7 +24,7 @@ def __init__(
             password=None, token=None, ignore_authrc=False,
             trust_all_ssl_certificates=False,
             auth_svc='https://ci.kbase.us/services/auth/api/legacy/KBase/Sessions/Login',
-            service_ver='release',
+            service_ver='beta',
```

Comment:
Do we want to do this?  Or is this just for testing?

Reply:
https://github.com/kbase/narrative/blob/develop/kbase-extension/static/kbase/js/widgets/function_output/kbaseReadsViewer.js uses it

Reply:
ok.  so we won't get rid of ReadsAPI.  it should get TLC then.  But that's beyond the scope of this module.  I removed the dependency on ReadsAPI for hipmer.